docs/verification.md

```diff
@@ -0,0 +1,201 @@
+        012b - e1 48 37 62 5c 39 56 bc-78 a6 84 d5 c3   .H7b\9V.x....
+        0138 - df 3d ea e4 75 80 07 a3-25 b9 c9 42 e6   .=..u...%..B.
+        0145 - 34 8e 49 22 20 4a 88 54-e3 e8 ed dd f0   4.I" J.T.....
+        0152 - 4b f4 e2 95 55 da a8 44-be 87 85 e6 d9   K...U..D.....
+        015f - 57 52 8f 97 b3 3a d3 d7-96 32 f9 22 20   WR...:...2." 
+        016c - 35 b2 b6 5b 9f 02 a8 bc-7d d2 f8 64 30   5..[....}..d0
+        0179 - d5 04 b1 c4 bb 2e 0c c8-bd 00 18 52 bb   ...........R.
+        0186 - 40 ad 84 6c 2d 68 22 20-4c 82 cf f1 63   @..l-h" L...c
+        0193 - 90 df b5 b4 3a 8b 0f bf-04 43 3e 52 0e   ....:....C>R.
+        01a0 - ef f6 d0 0e d3 c0 01 31-b1 8f 1b 68 82   .......1...h.
+        01ad - 74 22 20 ec 4c 65 15 56-3a 67 6a 41 1e   t" .Le.V:gjA.
+        01ba - 44 ad 06 b2 df 2d ff da-2c 03 77 87 ee   D....-..,.w..
+        01c7 - ba 00 c9 5b c3 b5 34 59-55 22 20 d6 30   ...[..4YU" .0
+        01d4 - 92 c2 27 78 05 dc b4 cb-36 1b ea 6e 09   ..'x....6..n.
+        01e1 - ac 7e d9 e9 e9 19 27 24-b8 f5 1e 57 e5   .~....'$...W.
+        01ee - 4b df 35 31 22 20 9e 04-00 66 df e5 f0   K.51" ...f...
+        01fb - 20 04 65 83 86 ac 66 cf-0b b6 ff e8 57    .e...f.....W
+        0208 - ed 71 cb 33 7c 7f 55 45-ec f4 55 8b 2a   .q.3|.UE..U.*
+        0215 - fe 01 0a fb 01 72 65 6b-6f 72 2e 73 69   .....rekor.si
+        0222 - 67 73 74 6f 72 65 2e 64-65 76 20 2d 20   gstore.dev - 
+        022f - 32 36 30 35 37 33 36 36-37 30 39 37 32   2605736670972
+        023c - 37 39 34 37 34 36 0a 31-34 30 33 33 37   794746.140337
+        0249 - 33 37 0a 50 31 2b 38 41-39 71 55 54 68   37.P1+8A9qUTh
+        0256 - 63 46 52 4b 6a 43 47 2b-6d 6e 62 49 52   cFRKjCG+mnbIR
+        0263 - 39 4f 57 5a 4c 42 79 2f-43 65 30 6b 39   9OWZLBy/Ce0k9
+        0270 - 4b 39 71 61 68 44 41 3d-0a 54 69 6d 65   K9qahDA=.Time
+        027d - 73 74 61 6d 70 3a 20 31-36 38 31 37 35   stamp: 168175
+        028a - 31 35 38 35 32 37 34 35-37 36 37 30 31   1585274576701
+        0297 - 0a 0a e2 80 94 20 72 65-6b 6f 72 2e 73   ..... rekor.s
+        02a4 - 69 67 73 74 6f 72 65 2e-64 65 76 20 77   igstore.dev w
+        02b1 - 4e 49 39 61 6a 42 45 41-69 42 31 56 4a   NI9ajBEAiB1VJ
+        02be - 48 46 6e 34 47 4e 63 32-65 38 65 42 78   HFn4GNc2e8eBx
+        02cb - 48 6f 4b 41 6c 56 6f 77-44 77 4a 51 72   HoKAlVowDwJQr
+        02d8 - 34 32 53 50 56 37 64 2f-6e 72 73 47 34   42SPV7d/nrsG4
+        02e5 - 77 49 67 4c 49 73 36 77-2b 59 75 39 42   wIgLIs6w+Yu9B
+        02f2 - 2f 35 2b 73 6b 6e 72 51-65 36 58 33 72   /5+sknrQe6X3r
+        02ff - 68 6e 6b 41 65 6a 6d 76-55 6d 4d 5a 5a   hnkAejmvUmMZZ
+        030c - 69 51 75 4d 53 49 59 3d-0a               iQuMSIY=.
+```
+
+These OIDs are defined by [Rekor](https://github.com/sigstore/rekor) and are
+used during verification to reconstruct the Rekor log entry and verify the
+commit signature.
+
+## Online Verification
+
+In online Rekor storage mode Gitsign will store the Git commit SHA in rekor
```

Comment:
I had to read this a few times to really grok the difference. IIUC:

1. "online" (old way): *after* creating the Git signature, do the Rekor upload (of the commit SHA). Verifiers use 
2. "offline" (new way): *while* creating the Git signature, do the Rekor upload (of the new sha256 hashed thingy) and stick the LogEntryAnon in the PKCS#7 signature

I wonder if it would be clearer to structure both sections as:

- what gets uploaded to Rekor
- how we sign
- how we verify

Reply:
Done.